draft-ietf-rtcweb-security-arch.xml

```diff
@@ -556,6 +556,162 @@
+        of an endpoint is bound to the fingerprints of the endpoint. This does not preclude the 
+        definition of alternative means of binding an assertion to the identity, but such means are 
+        outside the scope of this specification.
+      </t>
+      <t>
+	    The semantics of multiple 'identity' attributes within an offer or answer are undefined.
+	    Implementations SHOULD only include a single identity attribute in an
+	    offer or answer and relying parties MAY elect to ignore all but the first
+	    identity attribute.
+      </t>
+      <t>
+        <list style="hanging">
+        <t>
+          Name: identity
+        </t>
+        <t>
+          Value: identity-assertion
+        </t>
+        <t>
+          Usage Level: session
+        </t>
+        <t>
+          Charset Dependent: no
+        </t>
+        <t>
+          Default Value: N/A
+        </t>
+        </list>
+      </t>
+      <figure>
+      <artwork type="inline"><![CDATA[
+Syntax:
+
+  identity-assertion        = identity-assertion-value 
+                              *(SP identity-extension)
+  identity-assertion-value  = base64
+  identity-extension        = extension-name [ "=" extension-value ]
+  extension-name            = token
+  extension-value           = 1*(%x01-09 / %x0b-0c / %x0e-3a / %x3c-ff)
+                              ; byte-string from [RFC4566]
+
+  <ALPHA and DIGIT as defined in [RFC4566]>
+  <base64 as defined in [RFC4566]>
+  
+ Example:
+
+  a=identity:\
+    eyJpZHAiOnsiZG9tYWluIjoiZXhhbXBsZS5vcmciLCJwcm90b2NvbCI6ImJvZ3Vz\
+    In0sImFzc2VydGlvbiI6IntcImlkZW50aXR5XCI6XCJib2JAZXhhbXBsZS5vcmdc\
+    IixcImNvbnRlbnRzXCI6XCJhYmNkZWZnaGlqa2xtbm9wcXJzdHV2d3l6XCIsXCJz\
+    aWduYXR1cmVcIjpcIjAxMDIwMzA0MDUwNlwifSJ9
+    
+  Note that long lines in the example are folded to meet the column
+  width constraints of this document; the backslash ("\") at the end of
+  a line and the carriage return that follows shall be ignored.
+  
+      ]]></artwork>
+       </figure>
+       <t>
+         This specification does not define any extensions for the attribute
+       </t>
+       <t>
+         The identity-assertion value is a JSON <xref target="RFC8259"/> encoded string. The JSON object 
+         contains two keys: "assertion" and "idp". The <spanx style="verb">"assertion"</spanx> key value contains 
+         an opaque string that is consumed by the IdP. The <spanx style="verb">"idp"</spanx> key value contains a
+         dictionary with one or two further values that identify the IdP. See
+         <xref target="sec.request-assert"/> for more details.
+       </t>
+       <section title="Offer/Answer Considerations" anchor="sec.sdp-id-attr-oa">
+         <t>
+		   This section defines the SDP Offer/Answer <xref target="RFC6454"/> considerations for the SDP
+		   'identity' attribute.
+	     </t>
+         <t>
+           Within the section, 'initial offer' refers to the first offer, within an
```

Comment:
"Within *this* section.."? Otherwise it's not clear to me what "the section" refers to here.

Reply:
It was supposed to be "this". I will fix that.